tests/testTradeGhostValues.cc

```diff
@@ -13,132 +13,186 @@
 
 #include "catch.hpp"
 
+// function of periodicity nx, ny, nz
```

Comment:
this should not appear as a difference. please rebase

Reply:
Something is wrong even after the rebase, maybe because I copied it from your fork before it is merged.